### PR TITLE
block_resize: create the mount point before mount.

### PR DESCRIPTION
**Pull request by nickzhq**
block_resize: create the mount point before mount.
ID: 1994450  
Signed-off-by: Houqi (Nick) Zuo hzuo@redhat.com

### CODE REVIEW

**Comment by nickzhq** [comment]
@zhencliu @luckyh  Hello guys, please check this out! Thanks all!

File: qemu/tests/block_resize.py [hunk 1]
```diff
@@ -201,8 +201,11 @@ def get_md5_of_file(filename):
 
             if not utils_disk.is_mount(partition, dst=mpoint,
                                        fstype=fstype, session=session):
-                utils_disk.mount(partition, mpoint,
-                                 fstype=fstype, session=session)
+                session.cmd_output_safe("mkdir -p %s" % mpoint)
```
Comment (zhencliu) [inline qemu/tests/block_resize.py:204]: Only create the new mount point when the disk is changed, e.g. sdb1 -> sda1.
Here, if the disk is still sdb1, we don't need to create /mnt/sdb1, which was already present

**Comment by nickzhq** [comment]
@zhencliu @luckyh  Please check it out again! Thanks

**Comment by zhencliu** [comment]
hi @nickzhq , please paste the test result here.

**Comment by nickzhq** [comment]
(1/1) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.block_hotplug.block_scsi.fmt_raw.with_plug.with_block_resize.one_pci.q35: PASS (141.36 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

Hello @zhencliu  , please check it out

**Comment by zhencliu** [comment]
hi @qingwangrh , would you please review this patch, which fixed an issue you reported, thanks.

**Comment by qingwangrh** [comment]
python ConfigTest.py --testcase=block_hotplug.block_scsi.fmt_raw.default.with_plug.with_block_resize.one_pci.q35 --iothread_scheme=roundrobin --nr_iothreads=2 --platform=x86_64 --guestname=RHEL.9.0.0 --driveformat=virtio_scsi --nicmodel=virtio_net --imageformat=qcow2 --machines=q35  --customsparams="vm_mem_limit = 12G\nimage_aio=native" --clone=no --nrepeat=10

Test Passed 

LGTM

**Comment by zhencliu** [comment]
LGTM

**Comment by zhencliu** [comment]
hi @nickzhq , your commit msg is too long, would you please correct that?

    block_resize: create the mount point before mount.
    
    Root cause: According to the log, I found the issue happened at block_resize.py file, because the block_hotplug.py called the block_resize.py for sub-test .
    In the block_resize.py file, there was a utils_disk.mount() behavior which did NOT check the variant "mpoint". Once "mpoint" was NOT existence, utils_disk.mount() was FAILED definitely.
    For the solution, I have created a "make directory" behavior before the utils_disk.mount() behavior.
    
    Signed-off-by: Houqi (Nick) Zuo hzuo@redhat.com

**Comment by nickzhq** [comment]
@zhencliu  please check it out, thanks!

**Pull request merged**